CodeStarterCamp_Week3/person.swift

```diff
@@ -16,7 +16,25 @@ class Person {
         self.name = name
     }
     
-    func buy(coffee: Coffee, at coffeeShop: CoffeeShop) {
-        coffeeShop.takeOrderFor(coffee: coffee)
+    func order(_ coffee: Coffee, at coffeeShop: CoffeeShop) {
+        if let coffeeCost = coffeeShop.menuPan[coffee] {
+            let myCheckMoney = myMoneyCheckFor(by: coffeeCost)
+            
+            if myCheckMoney > 0 {
+                coffeeShop.takeOrderFor(coffee: coffee, by: self)
+            } else {
+                if myCheckMoney == 0 {
```

Comment:
이 부분은 합쳐서 작성할 수 있을 것 같습니다 💭
```suggestion
            } else if myCheckMoney == 0 {
```